.github/workflows/linux-ci-helper.sh

```diff
@@ -290,8 +293,11 @@ fi
 #-----------------------------------------------------------
 echo "${PRGNAME} [INFO] Set environment for configure options"
 
-echo "CXXFLAGS=${CXXFLAGS}"                   >> "${GITHUB_ENV}"
-echo "CONFIGURE_OPTIONS=${CONFIGURE_OPTIONS}" >> "${GITHUB_ENV}"
+cat << EOF > "${GITHUB_ENV}"
```

Comment:
The ${GITHUB_ENV} file has been overwritten.
Please change to addendum.
Is there a reason why you changed to Here Document?